distribution/ddf-common/src/main/resources/etc/log4j2.config.xml

```diff
@@ -104,14 +101,14 @@
 
     <Loggers>
         <Logger name="securityLogger" level="info" additivity="false">
-            <!--<AppenderRef ref="securityFailover"/>-->
+            <AppenderRef ref="securityFailover"/>
             <AppenderRef ref="securityMain"/>
```

Comment:
Great catch, this is a strange quirk of log4j2 configuration that did not make sense to me at first, it only occurred to me after a mailing list conversation. It seems it's only necessary to put the failover appender and not the primary appender.

Here's a gist I made in a simple hello world project I made that shows a working and non-working scenario:
https://gist.github.com/vinamartin/a52e28a900d1249ffccfe6dfe67fb533

Also seen here in their documentation:
https://logging.apache.org/log4j/2.x/manual/appenders.html#FailoverAppender